docs/source/intervention_models/targeted_bep_and_mms/index.rst

```diff
@@ -284,9 +288,19 @@ The code block below walks through how to implement the following considerations
 Assumptions and Limitations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- We assume that the birthweight shifts of maternal supplementation interventions are equal across the counterfactual unsupplemented birthweight exposure distribution. In reality the impact may be greater among the lower end of the birthweight distribution. Because the same shift in the birthweight among the lower end of the distribution is associated with a greater magnitude of mortality risk reduction than among the higher end of the distribution, we may underestimate the effect of the intervention. 
+
+- We assume that the birthweight shift for BEP reported by [Ota-et-al-2015]_ is relative to MMN, although it is actually relative to a reference group with mixed supplementation regimens. Due to the belief that the effect size of BEP may be underestimated (see discussion in the :ref:`BEP concept model document and manuscript <2017_concept_model_vivarium_gates_bep>`), this may not be a problematic assumption.
+
+- We do not consider effect modification by maternal anemia status.
```

Comment:
We might want to go beyond this limitation if we explicitly include the mothers in the sim, as @yongqx 's intervention model will.  To be re-examined when we bring together the mom effects and kid effects (is there any evidence you have seen that we can draw on)?